### PR TITLE
[REF] refactor bank_transfer payment

### PR DESCRIPTION
**Pull request by sebastienbeau**
Rename payment bank_transfer into payment_manual
Add acquired for check
Move demo data
Create a transaction in pending state for all manual payment

### CODE REVIEW

**Comment by sebastienbeau** [comment]
@sbidoul @lmignon @simahawk

**Comment by sebastienbeau** [comment]
After that I will push my test and the migration of shopinavder_quotation_payment in a separated PR

File: invader_payment_manual/demo/payment_demo.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo noupdate="1">
+
+<record id="payment_method_banktransfert" model="account.payment.mode">
```
Comment (simahawk) [inline invader_payment_manual/demo/payment_demo.xml:4]: remove the "t" please :)

**Comment by sebastienbeau** [comment]
There is still some pep 8 issue but black undo my change so...
But code is green

**Comment by sbidoul** [comment]
> black undo my change

Have you done `pre-commit install` in your local git clone of the project?

**Comment by sebastienbeau** [comment]
Yes, when I do the commit it, reformat it with more then 79 chars
The line with more then 79 is here : https://github.com/shopinvader/odoo-shopinvader-payment/pull/7/files#diff-d4ad96e3bb9da194cce28cc92b76e9c1R52

**Comment by lmignon** [comment]
@sebastienbeau To turn off autformat you need to add
```python
            # fmt: off
            res.append(
                {
                    "id": method.payment_mode_id.id,
                    "name": method.payment_mode_id.name,
                    "provider": method.payment_mode_id.payment_acquirer_id.provider,
                    "code": method.code,
                    "description": method.description,
                }
            )
            # fmt: on
```

and the code between `#fmt: off` and `fmt: on` will no more be formatted by black

**Pull request merged**